app/migrations/20250306124233-adds_project_table.cjs

```diff
@@ -20,7 +20,7 @@ module.exports = {
         onUpdate: "CASCADE",
       },
       city_count_limit: {
-        type: Sequelize.INTEGER,
+        type: Sequelize.BIGINT,
```

Comment:
This migration was likely run on the server already, meaning we should no longer change this migration file but rather add a new migration that changes the column type here.
This way we don't need to manually do/undo migrations on all envs and developer machines.